plugins/modules/proxmox_template.py

```diff
@@ -303,6 +360,9 @@ def main():
                 elif not proxmox.delete_template(node, storage, content_type, template, timeout):
                     module.fail_json(changed=False, msg='failed to delete template with volid=%s:%s/%s' % (storage, content_type, template))
 
+            if verify_checksum:
+                if proxmox.fetch_and_verify(node, storage, url, content_type, timeout, checksum, checksum_algorithm):
+                    module.exit_json(changed=True, msg="Checksum verified, template with volid=%s:%s/%s uploaded" % (storage, content_type, template))
```

Comment:
That means that if checksum verification is requested, the module will exit afer performing that verification and won't do anything else at all. Is that intentional?